src/Symfony/Component/Notifier/Bridge/GatewayApi/composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "symfony/gatewayapi-notifier",
+    "type": "symfony-bridge",
+    "description": "Symfony GatewayAPI Notifier Bridge",
+    "keywords": ["sms", "gatewayapi", "notifier"],
+    "homepage": "https://gatewayapi.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/http-client": "^4.3|^5.0",
+        "symfony/notifier": "^5.2"
+    },
+    "autoload": {
+        "psr-4": {
+            "Symfony\\Component\\Notifier\\Bridge\\GatewayApi\\": ""
+        },
+        "exclude-from-classmap": [
+            "/Tests/"
+        ]
+    },
+    "minimum-stability": "dev",
+    "extra": {
```

Comment:
Can you remove the `extra` section as we don't need it anymore.

Reply:
Done